src/config/constants/tokens.ts

```diff
@@ -597,6 +597,222 @@ const tokens = {
     decimals: 18,
     projectLink: 'https://dusk.network/',
   },
+  busd: {
+    symbol: 'BUSD',
```

Comment:
I think we should use the `QuoteToken` values here because we check against frequently.
```
farm.quoteToken.symbol === QuoteToken.BNB
```

Reply:
I wanted to remove this `QuoteToken` struct, because it doesn't make too much sens to have it, basically any token can be a quote token. Furthermore, we check against a lot in the farm for the apy, but like the pools, I'll change the apy algo to be more generic, so less check against the token name